diff --git a/components/viz/test/mock_display_client.h b/components/viz/test/mock_display_client.h
--- a/components/viz/test/mock_display_client.h
+++ b/components/viz/test/mock_display_client.h
@@ -32,9 +32,9 @@ class MockDisplayClient : public mojom::DisplayClient {
 #if BUILDFLAG(IS_APPLE)
   MOCK_METHOD1(OnDisplayReceivedCALayerParams, void(gfx::CALayerParams));
 #endif
-#if BUILDFLAG(IS_WIN)
   MOCK_METHOD1(CreateLayeredWindowUpdater,
                void(mojo::PendingReceiver<mojom::LayeredWindowUpdater>));
+#if BUILDFLAG(IS_WIN)
   MOCK_METHOD1(AddChildWindowToBrowser, void(gpu::SurfaceHandle child_window));
 #endif
 #if BUILDFLAG(IS_ANDROID)
